tests/integration/test_flask_sqlalchemy.py

```diff
@@ -39,9 +42,10 @@ def test_sqlalchemy(tracked_requests):
 
     assert response.status_int == 200
     assert response.text == "Hello from the DB!"
-    assert len(tracked_requests) == 1
+    # assert len(tracked_requests) == 1
```

Comment:
Is this check failing? looks like for the rest of the asserts to pass `tracked_requests` is at least not empty

Reply:
This was commented out for some testing I was doing, forgot to remove. Good catch.